File: uri_exercicios_em_python/begginner/even_or_odd.py
def even_or_odd(x):

    if x == 0:
        return "NULL"

    print("EVEN ", end="") if x % 2 == 0 and x != 0 else print("ODD ", end="")

    return "POSITIVE" if x > 0 else "NEGATIVE"


n = int(input())

for i in range(n):
    print(even_or_odd(int(input())))

File: beecrowd_exercicios_em_Cpp/beginner/even_or_odd.cpp
#include <iostream>

using namespace std;

void even_or_odd(){
    int x,y;
    cin >> x;

    for(int i = 0; i < x; i ++){
        cin >> y;
        if (y == 0) cout << "NULL" << endl;
        else{
            cout << (y % 2 == 0 ? "EVEN ": "ODD ");
            cout << (y > 0 ? "POSITIVE" : "NEGATIVE") << endl;
        }
    }
}

int main(){

    even_or_odd();

    return 0;
}

